src/pages/Order/Order.js

```diff
@@ -0,0 +1,217 @@
+                  type="text"
+                  name="email"
+                  placeholder="이메일을 입력해주세요."
+                  onChange={this.handleInput}
+                  value={email}
+                />
+              </div>
+              <div className="checkOutInput">
+                <label className="info">
+                  ZIP CODE
+                  <button className="findZip" onClick={this.handleOpenPost}>
+                    Find ZIP
+                  </button>
+                  {isDaumPost ? (
+                    <Address
+                      setAddress={this.setAddress}
+                      isDaumPost={this.state.isDaumPost}
+                    />
+                  ) : null}
+                </label>
+                <input
+                  className="orderInput"
+                  type="number"
+                  placeholder="우편번호를 입력해주세요."
+                  name="zip"
+                  onChange={this.handleInput}
+                  value={zip}
+                />
+              </div>
+              <div className="checkOutInput">
+                <label className="info">ADDRESS</label>
+                <input
+                  className="orderInput"
+                  type="text"
+                  name="address"
+                  placeholder="주소를 입력해주세요."
+                  onChange={this.handleInput}
+                  value={address}
+                />
+              </div>
+              <div className="btnLink">
+                <a className="goToCart" href="/cart">
+                  Return to cart
+                </a>
+                <button className="checkOutBtn" type="button">
+                  <span className="btnText">PROCEED TO CHECKOUT</span>
+                </button>
+              </div>
+            </div>
+            <ul className="orderList">
+              <div className="listTitle">CART LIST</div>
+              {cartList.map((cartlist, idx) => {
+                return (
+                  <Orderlist
+                    key={cartlist.option_id}
+                    idx={idx}
+                    cartList={cartlist}
+                  />
+                );
+              })}
+              <div className="couponContain">
+                <div className="couponArea">
+                  <input
+                    className="couponInput"
+                    type="text"
+                    name="coupon"
+                    placeholder="Gift card or discount code"
+                    onChange={this.handleInput}
+                    value={coupon}
+                  />
+                  <button className="couponBtn" onClick={this.sendCoupon}>
+                    APPLY
+                  </button>
+                </div>
+              </div>
+              <div className="shippingTotal">
+                <div className="subTotalArea">
+                  <div className="subTotalText">SUBTOTAL</div>
+                  <div className="totalPrice">{USDfomating(total)}</div>
```

Comment:
```suggestion
                  <div className="totalPrice">{formatToUSD(total)}</div>
```
대문자로 시작하는 건 컴포넌트입니다!

Reply:
> 대문자로 시작하는 건 컴포넌트입니다!

소문자로 변경완료했습니다.